### DISCUSSION TITLE
Which region are my pocketbase apps deployed?

### DISCUSSION THREAD

**Discussion by jonmaestas** [pockethost/pockethost#155] [Q&A]
Which region are my pocketbase apps deployed? 
I couldn't seem to find this anywhere. Also nice work on this project!

**Comment by benallfree** [comment]
PocketHost is powered by a Digital Ocean droplet. Currently `4 GB Memory / 80 GB Disk / SFO3 - Ubuntu 22.04 x64`.

The plan is to scale vertically first, then perhaps eventually horizontally. At that point, we may have an opportunity to choose the region where the instance (and data) lives.

**Comment by jonmaestas** [comment]
Got it. Thank you! I just now realized what **SF03** was. **San Fransisco**

I saw that before but only correlated that back to a Digital Ocean region just now.
